### ISSUE TITLE
Build failure on Android; No signature of method: java.lang.String.endsWithIgnoreCase()

### ISSUE THREAD

**Issue by samueljamesbell** [Instabug/Instabug-React-Native#969]
### Steps to Reproduce the Problem

1. Follow instructions to integrate React Native SDK on RN 
2. Try to launch simulator with `yarn run android`

### Expected Behavior

Simulator should launch - it works fine for RN on iOS.

### Actual Behavior

Get the following exception:

```
FAILURE: Build failed with an exception.

* Where:
Script '[redacted]/node_modules/instabug-reactnative/android/sourcemaps.gradle' line: 8

* What went wrong:
No signature of method: java.lang.String.endsWithIgnoreCase() is applicable for argument types: (String) values: [bundleReleaseJsAndAssets]

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 14s

error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* Where:
Script '[redacted]/node_modules/instabug-reactnative/android/sourcemaps.gradle' line: 8

* What went wrong:
No signature of method: java.lang.String.endsWithIgnoreCase() is applicable for argument types: (String) values: [bundleReleaseJsAndAssets]

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 14s

    at makeError ([redacted]/node_modules/execa/index.js:174:9)
    at [redacted]/node_modules/execa/index.js:278:16
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async runOnAllDevices ([redacted]/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/runOnAllDevices.js:109:5)
    at async Command.handleAction ([redacted]/node_modules/@react-native-community/cli/build/index.js:192:9)
info Run CLI with --verbose flag for more details.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Instabug integration code

```
import Instabug, { InvocationEvent } from 'instabug-reactnative';

Instabug.init({
  token: 'REDACTED',
  invocationEvents: [InvocationEvent.shake],
});
```

### SDK Version

11.10.0

### React Native, iOS and Android Versions

react-native 0.66.4
Build failure, targeting simulator Pixel 5 API 31

### Device Model

Pixel 5 API 31

**Comment by a7medev** [comment]
@samueljamesbell We were able to reproduce the issue from our side and created a snapshot with the fix which you can install using the command below:

```sh
npm install https://github.com/Instabug/Instabug-React-Native\#snapshot/11.10.0-MOB-12247
```

Please give it a try and let us know if it fixes your issue.

**Comment by a7medev** [comment]
@samueljamesbell This issue is now fixed in v11.12.0, please update to the latest version instead of using the snapshot 🙏🏼
